### ISSUE TITLE
Don't add julia bin dir to path

### ISSUE THREAD

**Issue by davidanthoff** [jupyter/repo2docker#609]
Would it make more sense to create a symlink as recommended in the julia [docs](https://julialang.org/downloads/platform.html#generic-binaries)? That generally seems less intrusive, and generally there seems to be a fair bit of stuff in the bin folder that doesn't need to be on the PATH.

**Comment by betatim** [comment]
Happy to follow the lead of someone who has experience with Julia and knows what makes things feel "Julia-thonic" (What is Pytohnic for Julia?). Something to look out for is "does it break things that currently work, even if they are 'against the proper way of doing things'". I think we should be conservative in terms of breaking things that people might be relying on.

**Comment by davidanthoff** [comment]
I'm pretty positive that this wouldn't break anything. Plus, we can only do this for the `Project.toml` case, not the `REQUIRE` case, where we don't have any backwards compat issues because no one is using this right now :)

**Comment by NHDaly** [comment]
Yeah I think this sounds good. That said, I also think that there was never anything in the `bin/` directory except the `julia` binary and in v0.6- the `julia-debug` binary. So i don't think this is much of a concern one way or the other. (And, to be fair, the [docs you pointed to](https://julialang.org/downloads/platform.html#generic-binaries) list both options as valid):

> To run Julia, you can do any of the following:
> - Create a symbolic link to julia inside a folder which is on your system `PATH`
> - Add Julia's `bin` folder to your system `PATH` environment variable
> - Invoke the `julia` executable by using its full path, as in `/bin/julia`

But i don't see any harm if you think the symlinks are better for some reason?

**Comment by davidanthoff** [comment]
I actually tried this, and it didn't work. For some reason it then couldn't find the libjulia anymore. I'm pretty sure one could get it to work _somehow_, but I think I won't try longer :)

So I'd say this is clearly very, very low priority, and we could almost certainly also just leave things the way they are.
